pkg/network/tls.go

```diff
@@ -1,48 +1,16 @@
 package network
 
 import (
-	"fmt"
-
 	"github.com/multiformats/go-multiaddr"
 )
 
-// There is implementation of TLS protocol for
-// github.com/multiformats/go-multiaddr library in this file.
-//
-// After addition TLS protocol via `multiaddr.AddProtocol` function
-// the library is ready to parse "tls" protocol with empty body, e.g.:
-//
-//		"/dns4/localhost/tcp/8080/tls"
-
 const (
 	tlsProtocolName = "tls"
-
-	// tlsProtocolCode is chosen according to its draft version's code in
-	// original multiaddr repository: https://github.com/multiformats/multicodec.
 	tlsProtocolCode = 0x01c0
```

Comment:
Unused?

Reply:
was used for searching for `tls` protocol in `TLSEnabled` method

but i missed that it is exported const in library, so deleted that and changed `TLSEnabled` method